src/test/kotlin/org/jetbrains/research/ml/coding/assistant/util/FileTestUtil.kt

```diff
@@ -18,4 +18,29 @@ object FileTestUtil {
         }
         return files.asSequence()
     }
+
+    fun getInAndOutFilesMap(folder: String): Map<File, File> {
+        val inFileRegEx = "in_\\d*.py".toRegex()
+        val inOutFileRegEx = "(in|out)_\\d*.(py|xml)".toRegex()
+        val (inFiles, outFiles) = getNestedFiles(folder)
+            .filter { it.isFile && inOutFileRegEx.containsMatchIn(it.name) }
+            .partition { inFileRegEx.containsMatchIn(it.name) }
+        if (inFiles.size != outFiles.size) {
+            throw IllegalArgumentException(
+                "Size of the list of in files does not equal size of the list of out files if the folder: $folder"
```

Comment:
equal **to**; **in** the folder

maybe it's better to write input/output files instead of in/out?

Reply:
I'm not sure it is a good idea to have too long names.

Reply:
I'm not talking about variable names; I'm about the error message where it is unclear what are 'in files' or 'out files' since they look like prepositions + nouns